test_team_sci.py

```diff
@@ -1,8 +1,35 @@
 import unittest
-from pii_scan import show_aggie_pride, analyze_text
-
+from pii_scan import analyze_text, show_aggie_pride
 
 class TestTeamSci(unittest.TestCase):
+
+    def test_aggie_pride(self):
+        """Test to make sure the Aggie Pride function works"""
+        self.assertEqual('Aggie Pride - Worldwide', show_aggie_pride())
+
+    def test_itin_detect(self):
+        """Test to make sure ITIN (US_ITIN) is detected"""
+
+        # Positive Testcase with a valid ITIN
+        results = analyze_text('My ITIN is 900701234')
+        print(results)
+        self.assertIn('US_ITIN', str(results))
+
+        # Positive Testcase with another valid ITIN
```

Comment:
You left this comment as a Positive Testcase when it is really a negative test case.  Please update this comment so it matches what the code does.